doc/tutorials/dnn/dnn_android/dnn_android.markdown

```diff
@@ -26,63 +26,8 @@ Tutorial was written for the following versions of corresponding software:
 
 - Download MobileNet object detection model from https://github.com/chuanqi305/MobileNet-SSD. We need a configuration file `MobileNetSSD_deploy.prototxt` and weights `MobileNetSSD_deploy.caffemodel`.
 
-## Create an empty Android Studio project
-- Open Android Studio. Start a new project. Let's call it `opencv_mobilenet`.
-![](1_start_new_project.png)
-
-- Keep default target settings.
-![](2_start_new_project.png)
-
-- Use "Empty Activity" template. Name activity as `MainActivity` with a
-corresponding layout `activity_main`.
-![](3_start_new_project.png)
-
-  ![](4_start_new_project.png)
-
-- Wait until a project was created. Go to `Run->Edit Configurations`.
-Choose `USB Device` as target device for runs.
-![](5_setup.png)
-Plug in your device and run the project. It should be installed and launched
-successfully before we'll go next.
-@note Read @ref tutorial_android_dev_intro in case of problems.
-
-![](6_run_empty_project.png)
-
-## Add OpenCV dependency
-
-- Go to `File->New->Import module` and provide a path to `unpacked_OpenCV_package/sdk/java`. The name of module detects automatically.
-Disable all features that Android Studio will suggest you on the next window.
-![](7_import_module.png)
-
-  ![](8_import_module.png)
-
-- Open two files:
-
-  1. `AndroidStudioProjects/opencv_mobilenet/app/build.gradle`
-
-  2. `AndroidStudioProjects/opencv_mobilenet/openCVLibrary330/build.gradle`
-
-  Copy both `compileSdkVersion` and `buildToolsVersion` from the first file to
-  the second one.
-
-  `compileSdkVersion 14` -> `compileSdkVersion 26`
-
-  `buildToolsVersion "25.0.0"` -> `buildToolsVersion "26.0.1"`
-
-- Make the project. There is no errors should be at this point.
-
-- Go to `File->Project Structure`. Add OpenCV module dependency.
-![](9_opencv_dependency.png)
-
-  ![](10_opencv_dependency.png)
-
-- Install once an appropriate OpenCV manager from `unpacked_OpenCV_package/apk`
-to target device.
-@code
-adb install OpenCV_3.3.0_Manager_3.30_armeabi-v7a.apk
-@endcode
-
-- Congratulations! We're ready now to make a sample using OpenCV.
+## Create an empty Android Studio project and add OpenCV dependency
+Use this tutorial to add OpenCV to your project: https://www.youtube.com/watch?v=bR7lL886-uc.
```

Comment:
Please add reference to previous pages rather than external video.

Reply:
added
![image](https://github.com/opencv/opencv/assets/22337800/31a1e88f-08a7-48a7-8f0c-7df7cf76401c)